docs/content/querying/post-aggregations.md

```diff
@@ -162,14 +162,35 @@ The format of the query JSON is as follows:
   "postAggregations" : [{
     "type"   : "arithmetic",
     "name"   : "average",
+    "fn"     : "/",
+    "fields" : [
+           { "type" : "fieldAccess", "name" : "tot", "fieldName" : "tot" },
+           { "type" : "fieldAccess", "name" : "rows", "fieldName" : "rows" }
+         ]
+  }]
+  ...
+}
+```
+
+
+```json
+{
+  ...
+  "aggregations" : [
+    { "type" : "count", "name" : "rows" },
+    { "type" : "doubleSum", "name" : "tot", "fieldName" : "total" }
+  ],
+  "postAggregations" : [{
+    "type"   : "arithmetic",
+    "name"   : "part_percentage",
     "fn"     : "*",
     "fields" : [
        { "type"   : "arithmetic",
-         "name"   : "div",
+         "name"   : "ratio",
          "fn"     : "/",
          "fields" : [
-           { "type" : "fieldAccess", "name" : "tot", "fieldName" : "tot" },
-           { "type" : "fieldAccess", "name" : "rows", "fieldName" : "rows" }
+           { "type" : "fieldAccess", "name" : "part", "fieldName" : "part" },
```

Comment:
```fieldName``` should be the name of an aggregator. Maybe you should add one more aggregator like ```{ "type" : "doubleSum", "name" : "part", "fieldName" : "part" }```.

Reply:
I see